portfolio/src/main/webapp/photo_gallery.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Photo Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <style>
+    body{
+        background-color: #a8d0e6;
+    }
+    div.gallery {
+        margin: 5px;
+        border: 3px solid;
+        float: left;
+        width: 200px;
+    }
+    div.gallery img {
+        width: 100%;
+        height: auto;
+    }
+    div.img {
+        padding: 15px;
+        text-align: center;
+    }
+</style>
+<body>
+
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_3112.JPG">
+    <img src="/images/IMG_3112.JPG" alt="Cinque Terre" width="600" height="400">
+  </a>
+  <div class="img">GHC 2019</div>
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_3053.JPG">
+    <img src="/images/IMG_3053.JPG" alt="Northern Lights" width="600" height="400">
+  </a>
+  <div class="img">"I see myself at Google"</div>
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_3052.JPG">
+    <img src="/images/IMG_3052.JPG" alt="Mountains" width="600" height="400">
+  </a>
+  <div class="img">The big "G"</div>
+</div>
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_2897.JPG">
+    <img src="/images/IMG_2897.JPG" alt="a" width="600" height="400">
```

Comment:
Again for good practice, let's give proper "alt" text.

This is pretty important at google because we strive to have very accessible products that can be used by people who either cannot see or have extremely poor vision and must rely on these strings to understand your UI better